app/pages/sections/order-items/associated-services/dashboard/ui-tests/general.ui.test.js

```diff
@@ -0,0 +1,169 @@
+import nock from 'nock';
+import { ClientFunction, Selector } from 'testcafe';
+import { extractInnerText } from 'buying-catalogue-library';
+import content from '../manifest.json';
+
+const pageUrl = 'http://localhost:1234/order/organisation/order-1/associated-services';
+
+const setCookies = ClientFunction(() => {
+  const cookieValue = JSON.stringify({
+    id: '88421113', name: 'Cool Dude', ordering: 'manage', primaryOrganisationId: 'org-id',
+  });
+
+  document.cookie = `fakeToken=${cookieValue}`;
+});
+
+const mocks = () => {
+  // nock(orderApiUrl)
+  //   .get('/api/v1/orders/order-1/order-items?catalogueItemType=AssociatedService')
+  //   .reply(200, []);
+
+  // nock(orderApiUrl)
+  //   .get('/api/v1/orders/order-1/sections/description')
+  //   .reply(200, { description: 'Some order' });
+};
+
+const pageSetup = async (withAuth = true) => {
+  if (withAuth) {
+    mocks();
+    await setCookies();
+  }
+};
+
+const getLocation = ClientFunction(() => document.location.href);
+
+fixture('Associated-services - Dashboard page - general')
+  .page('http://localhost:1234/order/some-fake-page')
+  .afterEach(async (t) => {
+    const isDone = nock.isDone();
+    if (!isDone) {
+      nock.cleanAll();
+    }
+
+    await t.expect(isDone).ok('Not all nock interceptors were used!');
+  });
+
+test('when user is not authenticated - should navigate to the identity server login page', async (t) => {
+  nock('http://identity-server')
+    .get('/login')
+    .reply(200);
+
+  await pageSetup(false);
+  await t.navigateTo(pageUrl);
+
+  await t
+    .expect(getLocation()).eql('http://identity-server/login');
+});
+
+test('should render associated-services page', async (t) => {
+  await pageSetup();
+  await t.navigateTo(pageUrl);
+  const page = Selector('[data-test-id="associated-services-page"]');
+
+  await t
+    .expect(page.exists).ok();
+});
+
+// TODO Add when nocks are added
+// test('should navigate to /organisation/order-1 when click on backLink', async (t) => {
+//   await pageSetup();
+//   await t.navigateTo(pageUrl);
+
+//   const goBackLink = Selector('[data-test-id="go-back-link"] a');
+
+//   await t
+//     .expect(goBackLink.exists).ok()
+//     .click(goBackLink)
+//     .expect(getLocation()).eql('http://localhost:1234/order/organisation/order-1');

```

Comment:
instead of clicking on the backlink that causes us issue just check the href attribute is correct. Something like 
```
.expect(goBackLink.getAttribute('href')).eql('/order/organisation/order-1/additional-services/select/additional-service/price/recipient');
```

Reply:
Yeah we got the same issue. Callum just mentioned this, good shout 👍